flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveSourceBuilder.java

```diff
@@ -242,21 +249,33 @@ public <T> HiveSource<T> buildWithBulkFormat(BulkFormat<T, HiveSourceSplit> bulk
             hiveTablePartitionBytes = HivePartitionUtils.serializeHiveTablePartition(partitions);
         }
 
-        return new HiveSource<>(
-                new Path[1],
-                new HiveSourceFileEnumerator.Provider(
-                        hiveTablePartitionBytes, new JobConfWrapper(jobConf)),
-                splitAssigner,
-                bulkFormat,
-                continuousSourceSettings,
-                jobConf,
-                tablePath,
-                partitionKeys,
-                hiveVersion,
-                dynamicFilterPartitionKeys,
-                hiveTablePartitionBytes,
-                fetcher,
-                fetcherContext);
+        DefaultSourceLineageVertex vertex =
```

Comment:
I like the idea of adding a default source vertex and a default vertex. I wonder if there are any Hive specifis we could  add to the vertex that would warrant not using the default.